server/middleware/response.go

```diff
@@ -0,0 +1,16 @@
+package middleware
+
+import (
+	"net/http"
+
+	"github.com/zenazn/goji/web"
+)
+
+// Set response as JSON
+func SetJSONResponse(c *web.C, h http.Handler) http.Handler {
```

Comment:
for consistency, perhaps we could put it in `SetHeaders` with all the other headers?
https://github.com/drone/drone/blob/master/server/middleware/header.go